xbmc/addons/APIVersions.h

```diff
@@ -0,0 +1,68 @@
+      {"xbmc.metadata", AddonVersion("2.1.0"), AddonVersion("1.0")},
+      {"xbmc.python", AddonVersion("2.25.0"), AddonVersion("2.1.0")},
+      {"xbmc.webinterface", AddonVersion("1.0.0"), AddonVersion("1.0.0")},
+      {"kodi.resource", AddonVersion("1.0.0"), AddonVersion("1.0.0")},
+      {ADDON_GLOBAL_VERSION_MAIN_XML_ID, AddonVersion(ADDON_GLOBAL_VERSION_MAIN), AddonVersion(ADDON_GLOBAL_VERSION_MAIN_MIN)},
+      {ADDON_GLOBAL_VERSION_GENERAL_XML_ID, AddonVersion(ADDON_GLOBAL_VERSION_GENERAL), AddonVersion(ADDON_GLOBAL_VERSION_GENERAL_MIN)},
+      {ADDON_GLOBAL_VERSION_GUI_XML_ID, AddonVersion(ADDON_GLOBAL_VERSION_GUI), AddonVersion(ADDON_GLOBAL_VERSION_GUI_MIN)},
+      {ADDON_GLOBAL_VERSION_AUDIOENGINE_XML_ID, AddonVersion(ADDON_GLOBAL_VERSION_AUDIOENGINE), AddonVersion(ADDON_GLOBAL_VERSION_AUDIOENGINE_MIN)},
+      {ADDON_GLOBAL_VERSION_FILESYSTEM_XML_ID, AddonVersion(ADDON_GLOBAL_VERSION_FILESYSTEM), AddonVersion(ADDON_GLOBAL_VERSION_FILESYSTEM_MIN)},
+      {ADDON_GLOBAL_VERSION_NETWORK_XML_ID, AddonVersion(ADDON_GLOBAL_VERSION_NETWORK), AddonVersion(ADDON_GLOBAL_VERSION_NETWORK_MIN)},
+      {ADDON_INSTANCE_VERSION_ADSP_XML_ID, AddonVersion(ADDON_INSTANCE_VERSION_ADSP), AddonVersion(ADDON_INSTANCE_VERSION_ADSP_MIN)},
+      {ADDON_INSTANCE_VERSION_AUDIODECODER_XML_ID, AddonVersion(ADDON_INSTANCE_VERSION_AUDIODECODER), AddonVersion(ADDON_INSTANCE_VERSION_AUDIODECODER_MIN)},
+      {ADDON_INSTANCE_VERSION_AUDIOENCODER_XML_ID, AddonVersion(ADDON_INSTANCE_VERSION_AUDIOENCODER), AddonVersion(ADDON_INSTANCE_VERSION_AUDIOENCODER_MIN)},
+      {ADDON_INSTANCE_VERSION_GAME_XML_ID, AddonVersion(ADDON_INSTANCE_VERSION_GAME), AddonVersion(ADDON_INSTANCE_VERSION_GAME_MIN)},
+      {ADDON_INSTANCE_VERSION_IMAGEDECODER_XML_ID, AddonVersion(ADDON_INSTANCE_VERSION_IMAGEDECODER), AddonVersion(ADDON_INSTANCE_VERSION_IMAGEDECODER_MIN)},
+      {ADDON_INSTANCE_VERSION_INPUTSTREAM_XML_ID, AddonVersion(ADDON_INSTANCE_VERSION_INPUTSTREAM), AddonVersion(ADDON_INSTANCE_VERSION_INPUTSTREAM_MIN)},
+      {ADDON_INSTANCE_VERSION_PERIPHERAL_XML_ID, AddonVersion(ADDON_INSTANCE_VERSION_PERIPHERAL), AddonVersion(ADDON_INSTANCE_VERSION_PERIPHERAL_MIN)},
+      {ADDON_INSTANCE_VERSION_PVR_XML_ID, AddonVersion(ADDON_INSTANCE_VERSION_PVR), AddonVersion(ADDON_INSTANCE_VERSION_PVR_MIN)},
+      {ADDON_INSTANCE_VERSION_SCREENSAVER_XML_ID, AddonVersion(ADDON_INSTANCE_VERSION_SCREENSAVER), AddonVersion(ADDON_INSTANCE_VERSION_SCREENSAVER_MIN)},
+      {ADDON_INSTANCE_VERSION_VFS_XML_ID, AddonVersion(ADDON_INSTANCE_VERSION_VFS), AddonVersion(ADDON_INSTANCE_VERSION_VFS_MIN)},
+      {ADDON_INSTANCE_VERSION_VISUALIZATION_XML_ID, AddonVersion(ADDON_INSTANCE_VERSION_VISUALIZATION), AddonVersion(ADDON_INSTANCE_VERSION_VISUALIZATION_MIN)},
+      {ADDON_INSTANCE_VERSION_VIDEOCODEC_XML_ID, AddonVersion(ADDON_INSTANCE_VERSION_VIDEOCODEC), AddonVersion(ADDON_INSTANCE_VERSION_VIDEOCODEC_MIN)},
```

Comment:
@AlwinEsch If you are around, what do you want me to do with these? Should I remove addons/kodi-addon-dev-kit/include/kodi/versions.h or keep it like this?